packages/accordion/src/Accordion.ts

```diff
@@ -30,7 +32,7 @@ import styles from './accordion.css.js';
  * @element sp-accordion
  * @slot - The sp-accordion-item children to display.
  */
-export class Accordion extends SpectrumElement {
+export class Accordion extends SizedMixin(SpectrumElement) {
```

Comment:
We don't want to apply a size by default, so let's use the second argument noDefaultSize so it doesn't assume it needs to apply size m by default. This will improve the performance of the component.